conftest.py

```diff
@@ -229,13 +229,15 @@ def auto_benchmark_time(benchmark_time):
 def benchmark_memory(test_run_benchmark):
     @contextlib.contextmanager
     def _benchmark_memory(client):
-        sampler = MemorySampler()
-        label = uuid.uuid4().hex[:8]
-        with sampler.sample(label, client=client, measure="process"):
+        if test_run_benchmark:
             yield
+        else:
+            sampler = MemorySampler()
+            label = uuid.uuid4().hex[:8]
+            with sampler.sample(label, client=client, measure="process"):
+                yield
 
-        df = sampler.to_pandas()
-        if test_run_benchmark:
```

Comment:
Technically, this line prevented `benchmark_memory` from raising, but I've adopted the same pattern we use in the other fixtures to reduce cognitive load. @ian-r-rose